site/content/integrate/slash-commands/_index.md

```diff
@@ -156,46 +62,56 @@ Content-Length: 696
 }
 ```
 
-#### What if my slash command takes time to build a response?
+### What if my slash command takes time to build a response?
 
 Reply immediately with an `ephemeral` message to confirm response of the command, and then use the `response_url` to send up to five additional messages within a 30-minute time period from the original command invocation.
 
-#### Why does my slash command fail to connect to `localhost`?
+### Why does my slash command fail to connect to `localhost`?
 
 By default, Mattermost prohibits outgoing connections that resolve to certain common IP ranges, including the loopback (`127.0.0.0/8`) and various private-use subnets.
 
-During development, you may override this behaviour by setting `ServiceSettings.AllowedUntrustedInternalConnections` to `"127.0.0.0/8"` in your `config.json` or via System Console **Advanced > Developer**. See the [admin guide's notes](https://docs.mattermost.com/configure/configuration-settings.html#allow-untrusted-internal-connections-to) for more details.
+During development, you may override this behaviour by setting `ServiceSettings.AllowedUntrustedInternalConnections` to `"127.0.0.0/8"` in your `config.json` or via **System Console > Advanced > Developer**. See the [admin guide's notes](https://docs.mattermost.com/configure/environment-configuration-settings.html#allow-untrusted-internal-connections) for more details.
```

Comment:
One more instance of "admin guide" to update. Let's update this one to "See the configuration settings documentation for more details" that includes the link to the product docs.